boskos/client/client.go

```diff
@@ -17,17 +17,26 @@ limitations under the License.
 package client
 
 import (
+	"bytes"
 	"encoding/json"
+	"flag"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"net/http"
 	"sync"
 	"time"
 
 	"k8s.io/test-infra/boskos/common"
+	"strings"
 )
 
-// Public Boskos client object
+var (
+	// BoskosURL defines URL where boskos HTTP server is running
+	BoskosURL = flag.String("boskos-url", "http://boskos", "Boskos URL")
```

Comment:
packages really shouldn't export flags :(

Reply:
Just out of curiosity, what's the reason ? Is there a guideline you can point me to ?

Reply:
This registers a global flag in a flat namespace that the importing binary isn't aware of and has no control over. Usage really ought to be defined by commands, not libraries / sub-packages.